### DISCUSSION TITLE
What would be the best method to implement VoiceOver for TextBox on iOS?

### DISCUSSION THREAD

**Discussion by doum-vigeant-nventive** [unoplatform/uno#12679] [Q&A] [answered]
I'm trying to improve the screen reader support for the UnoApplicationTemplate. The behavior for `TextBox` on iOS is not up to standards for accessibility. 

Right now in the app template, when focusing a `TextBox`, VoiceOver reads it like so:

- If `TextBox` is empty:
  - `PlaceholderText`
- If `TextBox` is filled:
  - `PlaceholderText`

In comparison, when you use the form to create a new contact on the iOS built-in app Contacts, VoiceOver reads it that way:

- If `TextBox` is empty:
  - `Label`
  - `PlaceholderText`
- If `TextBox` is filled:
  - `Label`
  - Text input by user

What doesn't work is:
- Using `AutomationProperties.Name` in xaml
- Using `AutomationProperties.HelpText` in xaml

Do you have ideas on what would be the best way to support VoiceOver for `TextBox`?

**Comment by jeromelaban** [comment, marked as answer]
This looks like a missing feature. Let's track in https://github.com/unoplatform/uno/issues/12684.
